ci-operator/config/medik8s/common/new-ocp-version.sh

```
@@ -43,7 +48,7 @@ for repo in */ ; do
   for branch in $branches; do
     echo "branch: $branch"
     # find newest OCP version
-    version=$(ls | grep .yaml | grep "__" | grep medik8s-$repo-$branch | sed -r 's#^.*__(.*)\.yaml$#\1#g' | sort | tail -1)
+    version=$(ls | grep .yaml | grep "__" | grep medik8s-$repo-$branch | sed -r 's#^.*__(.*)\.yaml$#\1#g' | sort -V | tail -1)
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)